### PR TITLE
Clean up Styling and Formatting

### PR DESCRIPTION
**Pull request by kaylawoodbury**
### User Story
```
As a developer team
In order to deliver a presentable app to our client
We need to update our final styling and formatting
```
## Changes proposed in this pull request:
* Add spinning icon when signing in
* Add phone number to Pong Details
* Add functionality to checkboxes so they can check on/off
* Address most warnings/error messages from inspect console
* Update margins to be more consistent
* Updates styling on buttons
* Updates order of fields on profile
* Updates date field for new trip to be split into two fields

## What I have learned working on this feature:
* How to implement a spinning icon
* Sometimes the small tweaks can be more difficult than larger changes

## Screenshots (Client)
Screenshot 2020-04-24 at 17 33 18
Screenshot 2020-04-24 at 17 35 25
Screenshot 2020-04-24 at 17 35 52

### CODE REVIEW

File: screens/PongToPingDetails.jsx [hunk 1]
```diff
@@ -58,11 +62,14 @@ const PongToPingDetails = (
     })
   }
 
+
```
Comment (kfrostare) [inline screens/PongToPingDetails.jsx]: Two extra spaces here.

File: cypress/integration/visitorCanSeeListOfUpcomingTrips.feature.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ describe("Visitor can see list of all upcoming active trips", () => {
   });
   it("unsuccessfully", () => {
```
Comment (kfrostare) [inline cypress/integration/visitorCanSeeListOfUpcomingTrips.feature.js:56]: a space here?

File: screens/PongToPingDetails.jsx (hunk 1 above)
Comment (kaylawoodbury) [reply screens/PongToPingDetails.jsx]: removed

File: cypress/integration/visitorCanSeeListOfUpcomingTrips.feature.js (hunk 2 above)
Comment (kaylawoodbury) [reply cypress/integration/visitorCanSeeListOfUpcomingTrips.feature.js:56]: added

**Review by pgaunitz — APPROVED** [review]
:+1:

**Comment by kfrostare** [comment]
Looks good to me <3

**Pull request merged**